client-mr/src/main/java/org/apache/hadoop/mapreduce/task/reduce/RssRemoteMergeManagerImpl.java

```diff
@@ -144,8 +144,9 @@ public RssRemoteMergeManagerImpl(String appId, TaskAttemptID reduceId, JobConf j
     try {
       remoteConf.setInt("dfs.replication", replication);
       remoteConf.setInt("dfs.client.block.write.retries", retries); // origin=3
-      this.remoteFS = ShuffleStorageUtils.getFileSystemForPath(new Path(basePath), remoteConf);
-    } catch (IOException e) {
+      HadoopAccessorProvider.init();
+      this.remoteFS = HadoopAccessorProvider.getFileSystem(new Path(basePath), remoteConf);
```

Comment:
This part will write spill data to hdfs. 
One question:
If Hdfs use kerberos, it's necessary to use kerberos in Spark or MR client?

Reply:
No need. The auth of HDFS should be managed by computing framework.

Reply:
Could the client use this `fs` to write data to secured HDFS?

Reply:
Yes.The spark/mr will manage the credentials of HDFS delegation token for all executors, so there is no need to relogin in shuffle manager client of uniffle.